system/Filters/Honeypot.php

```diff
@@ -52,5 +54,7 @@ public function before(RequestInterface $request, $arguments = null)
     public function after(RequestInterface $request, ResponseInterface $response, $arguments = null)
     {
         service('honeypot')->attachHoneypot($response);
+
+        return $response;
```

Comment:
same question, here.

Reply:
same with https://github.com/codeigniter4/CodeIgniter4/pull/9311#pullrequestreview-2492428970 , the response changed, so I guess returning null should not happen.